docs/sources/tempo/release-notes/v2-2.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Version 2.2 release notes
+menuTitle: V2.2
+description: Release notes for Grafana Tempo 2.2
+weight: 50
+---
+
+# Version 2.2 release notes
+
+The Tempo team is pleased to announce the release of Tempo 2.2.
+
+This release gives you:
+
+- Major additions to the TraceQL language: structural operators (descendant, child, sibling), results grouped by attribute (`by()`), and 3 new intrinsic attributes.
+- Faster TraceQL results, thanks to a streaming endpoint that returns partial results as the query executes as well as a multitude of performance-related improvements.
+- An experimental metrics-summary API that returns RED metrics for recently received spans grouped by your attribute of choice.
+
+Tempo 2.2 makes vParquet2, a Parquet version designed to be more compatible with other Parquet implementations, the default block format. This block format is required for using structural operators and improves query performance relative to previous formats.
+
+Read the [Tempo 2.2 blog post](/blog/NEED BLOG LINK) for more examples and details about these improvements.
```

Comment:
:chicken: and :egg: here. i need this backported to 2.2 so i can link to it from the blog. i think we should get this in the 2.2 release branch first then we can go back and add the correct blog link later

Reply:
I can give you the link for the rel notes: https://grafana.com/docs/tempo/latest/release-notes/v2-2/

Reply:
added to blogpost!